### PR TITLE
Many changes to prclz

### PR DESCRIPTION
**Pull request by mmerrittsmith**
A hopefully complete but likely not list: 

.shp.csv should just be .csv (created by blocks)
complexity.py to _complexity.py to address the CLI issues
Path to output should be a file not a folder in complexity.py
Add more documentation w/ file paths and endings and stuff
Add _extract.py
Be more expressive about creating files

### CODE REVIEW

**Comment by mmerrittsmith** [comment]
I should also note that I will be continuing to edit this PR-- my work is a bit scattered between Midway and local, so I'm not 100% certain I've implemented the changes the way I wanted to in this PR.

**Comment by mmerrittsmith** [comment]
Still need to add an override for the user input on download and make sure that I've changed the manifest and all that to get the extract script and osmconf.ini.  Also an examples folder with the files in there alongside a README.md as a worked examples, one for local one for Midway (like create an sbatch script, which will run the whole thing for all countries). Need to say stuff like install GDAL, install packages X and Y, etc.

**Comment by satejsoman** [comment]
i like the direction these changes are going! the name "many changes" is a good tell that this might be more effective to implement and review as several small PRs (though I get that the tradeoff is that it's harder to keep track of what's ported over from Midway vs your local branch, etc)

**Comment by nmarchio** [comment]
@satejsoman  since these are pretty small changes and mostly all part of a common CLI issue, do you think it would be alright to merge as oppose to splitting up into many prs. @mmerrittsmith has test on Midway and is now testing locally -- if it all runs my suggestion would be to merge (this is blocking right now and we wanna just move past this etl debugging) and none of the changes affect underlying computation.

**Comment by satejsoman** [comment]
@nmarchio yeah, that's fine. probably don't necessarily need my approval once it's out of draft mode.

**Comment by nmarchio** [comment]
Thanks @satejsoman . @mmerrittsmith after you address my comments, this should be LGTM. Thanks

File: prclz/etl/_download.py [hunk 1]
```diff
@@ -26,16 +26,14 @@ def build_data_dir(root: str, additional: Optional[Sequence[str]] = None, permis
...
     else:
-        info('Downloading will create the following repositories:')
+        info('Downloading will create the following repositories if they do not already exist:')
         for dir_path in data_paths.values():
-            info(f'{dir_path}')
+            info(f'{dir_path.resolve()}')
         user_response = input('Do you wish to proceed with download to the specified folder? Please enter "y" or "n".')
         while user_response.lower() not in ['y', 'n']:
-            user_repsonse = input('Please enter "y" or "n".')
+            user_repsonse = input('Please enter "y" or "n"')
```
Comment (satejsoman) [inline prclz/etl/_download.py:34]: typo in 'response'

Comment (satejsoman) [reply prclz/etl/_download.py:34]: might be worth doing a run with `mypy` or other linters to catch this sort of thing

**Pull request merged**